### PR TITLE
Fix UI/therapy op 535

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: cypress/integration/login_activity_spec.js [hunk 1]
```diff
@@ -92,4 +92,42 @@ describe("LoginActivity spec", () => {
...
+    if (cy.get("[class=appHeader__identified__trigger]")) {
+      cy.get("[class=appHeader__identified__trigger]").click();
+      cy.get("[id=signout_icon]").click();
+    } else {
+      cy.get("[id=signout_icon]").click();
+    }
+    //check if the confirmation dialog is rendered
+    cy.get(".MuiDialog-paper");
+    //dismiss the confirmation dialog
+    cy.get(".reset_button button").click();
```
Comment (axelpezzo) [inline cypress/integration/login_activity_spec.js:107]: I think that should be more accurate have .MuiDialog-paper .reset_button button in order to exclude the others .reset_button button not in the scope.

File: cypress/integration/login_activity_spec.js [hunk 2]
```diff
@@ -92,4 +92,42 @@ describe("LoginActivity spec", () => {
...
+    //If the reduce menu icon is present, open the menu page and click on signout icon
+    // else click directly on the signout icon
+    if (cy.get("[class=appHeader__identified__trigger]")) {
+      cy.get("[class=appHeader__identified__trigger]").click();
+      cy.get("[id=signout_icon]").click();
+    } else {
+      cy.get("[id=signout_icon]").click();
+    }
+    //validate the signout action
+    cy.get(".return_button button").click();
```
Comment (axelpezzo) [inline cypress/integration/login_activity_spec.js:127]: as before

File: src/components/accessories/appHeader/AppHeader.tsx [hunk 3]
```diff
@@ -73,11 +87,38 @@ const AppHeader: FunctionComponent<TProps> = ({ breadcrumbMap }) => {
...
             <div className="appHeader__nav__item">{t("nav.ward")}</div>
             <div className="appHeader__nav__item">{t("nav.billing")}</div>
+            <div className="appHeader__nav__item">
+              <Tooltip title={t("login.signout")!} aria-label="sign out">
+                <ExitToAppIcon
+                  id="signout_icon"
+                  onClick={() => setOpenLogoutConfirmation(true)}
+                />
+              </Tooltip>
+            </div>
```
Comment (axelpezzo) [inline src/components/accessories/appHeader/AppHeader.tsx:97]: Shouldn't be a appHeader__nav__item. This classes should be used in order to render the main menu.
Try to create a new div, maybe placing it near the languageSwitcher component and render it in this way (take a look at the span element):

```
<div className="userInfo__toolbar">
  <Tooltip title={t("login.signout")!} aria-label="sign out">
    <span>Welcome, <strong>{username}</strong></span>
    <ExitToAppIcon
      className="userInfo__toolbar_icon"
      id="signout_icon"
      onClick={() => setOpenLogoutConfirmation(true)}
    />
  </Tooltip>
</div>

```

File: src/components/accessories/appHeader/consts.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export const LOGIN_URL = "/login";
```
Comment (axelpezzo) [inline src/components/accessories/appHeader/consts.ts:1]: Is this constant used anywhere?

**Review by axelpezzo — CHANGES_REQUESTED** [review]
I leave you some comments on AppHeader and on Cypress test, just small fixes in order to improve the UI and the test.

File: cypress/integration/login_activity_spec.js (hunk 1 above)
Comment (tsognong) [reply cypress/integration/login_activity_spec.js:107]: OK

Comment (tsognong) [reply cypress/integration/login_activity_spec.js:107]: Ok, I change it. thanks

File: cypress/integration/login_activity_spec.js (hunk 2 above)
Comment (tsognong) [reply cypress/integration/login_activity_spec.js:127]: Ok, I change it. thanks

File: src/components/accessories/appHeader/consts.ts (hunk 4 above)
Comment (tsognong) [reply src/components/accessories/appHeader/consts.ts:1]: I removed it.

**Comment by tsognong** [comment]
> I leave you some comments on AppHeader and on Cypress test, just small fixes in order to improve the UI and the test.

thanks for the comments!!!

File: src/components/accessories/appHeader/AppHeader.tsx (hunk 3 above)
Comment (tsognong) [reply src/components/accessories/appHeader/AppHeader.tsx:97]: Y're right.

**Pull request merged**